ansible/roles/nameindex/tasks/main.yml

```diff
@@ -48,21 +48,20 @@
   tags:
     - "nameindex"
 
-## Ensure that data_dir/lucene exists as a directory
-
+## Ensure that data_dir/lucene exists as a directory (ALA and not-ALA)
```

Comment:
Can you split the nameindex changes out into a separate pull request please?

Reply:
Right, I'll do it

Reply:
thanks @JeGoi